From 2650ac40163f2e79b3679f943681efacb6b7363c Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Mon, 5 Feb 2018 15:12:58 +0800
Subject: [PATCH 3495/5242] MLK-17230-2: CI_PI: add power domain names for
 CI_PI ss

commit  cbb4eb25371bab9ff770fbf02ee41de346e481ee from
https://source.codeaurora.org/external/imx/linux-imx.git

Add power domain macro names for CI_PI subsystem.

Reviewed-by: Sandor.Yu <sandor.yu@nxp.com>
Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
(cherry picked from commit fd8318f4455ceafda963681ce05effd0ad81d714)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/dt-bindings/soc/imx8_pd.h |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/include/dt-bindings/soc/imx8_pd.h b/include/dt-bindings/soc/imx8_pd.h
index 34fd2e9..b7afa19 100644
--- a/include/dt-bindings/soc/imx8_pd.h
+++ b/include/dt-bindings/soc/imx8_pd.h
@@ -1,6 +1,6 @@
 /*
  * Copyright (C) 2016 Freescale Semiconductor, Inc.
- * Copyright 2017 NXP
+ * Copyright 2017-2018 NXP
  *
  * This program is free software; you can redistribute it and/or
  * modify it under the terms of the GNU General Public License
@@ -182,6 +182,11 @@
 #define PD_MIPI_CSI1_PWM_0          mipi_csi1_pwm
 #define PD_MIPI_CSI1_I2C0           mipi_csi1_i2c0
 
+#define PD_PARALLEL_CSI             parallel_csi_power_domain
+#define PD_PARALLEL_CSI_I2C         parallel_csi_i2c
+#define PD_PARALLEL_CSI_PWM         parallel_csi_pwm
+#define PD_PARALLEL_CSI_PLL         parallel_csi_pll
+
 #define PD_HDMI                     hdmi_power_domain
 #define PD_HDMI_I2C_0               hdmi_i2c
 #define PD_HDMI_PWM_0               hdmi_pwm
-- 
1.7.9.5

